sources/views/activities.js

```diff
@@ -0,0 +1,86 @@
+import {JetView} from "webix-jet";
+import {activities} from "../models/activities";
+import {activitiesTypes} from "../models/activitiesTypes";
+import {contacts} from "../models/contacts";
+import activitiesForm from "./activitiesForm";
+
+export default class Activities extends JetView {
+	config() {
+		const button = {
+			css: "activities_button",
+			cols: [
+				{},
+				{
+					view: "button",
+					localId: "addButton",
+					value: "Add",
+					css: "webix_primary",
+					width: 150,
+					label: "Add activity",
+					type: "icon",
+					icon: "wxi-plus-circle",
+					click: () => this.showForm()
+				}
+			]
+		};
+
+		const table = {
+			view: "datatable",
+			editable: true,
+			select: true,
+			localId: "activitiesTable",
+			scroll: "auto",
+			columns: [
+				{id: "State", header: "", template: "{common.checkbox()}", checkValue: "Close", uncheckValue: "Open", width: 50},
+				{id: "TypeID", header: ["Activity type", {content: "selectFilter"}], collection: activitiesTypes, sort: "string"},
```

Comment:
@HelgaListopad I'm confused, sorting is performed by field and works good, but may be can sort alphabetically? Now it sorted Chat, Phone Call, Lunch nad vise versa what is not alphabetically

Reply:
Names of activity types are fetched from the separate collection (`activitiesTypes`) and rendered in the template, while the `sort:"string"` sorts the real data stored in the table (i.e. IDs of activity types). 
The issue can be resolved by setting the type:"text" (described in [docs](https://docs.webix.com/datatable__sorting.html#sortingbyaclickontheheader) among others)

Reply:
Changed sorting type to text